src/components/image/Image.props.ts

```diff
@@ -0,0 +1,40 @@
+import { TShapeBorderRadius } from '../../foundations/shape';
+
+import TImageSize from './Image.types';
+
+export type PImage = {
+    /**
+     * the src of the image
+     * */
+    source?: string;
+    /**
+     * whether the image is full sized or auto
+     * @default auto
+     * */
+    size?: TImageSize;
+    /**
+     * the shape that the image is cropped in
+     * @default 4px
+     * */
+    radius?: TShapeBorderRadius;
+    /**
+     * image width
+     * */
+    width?: number;
+    /**
+     * image height
+     * */
+    height?: number;
+    /**
+     * image alt
+     * */
+    alt?: string;
+    /**
+     * whether the image is a thumbnail
+     * */
+    thumbnail?: boolean;
+    /**
+     * provided custom class
+     * */
+    className?: string;
```

Comment:
nit: do we really need custom classes?

Reply:
I would avoid it for images but I feel like we should give the user that option? not sure 🤷